auth/src/main/java/com/redhat/lightblue/rest/auth/ldap/LightblueLdapRoleProvider.java

```diff
@@ -43,10 +40,10 @@
     InitialLdapContextProvider ldapContextProvider;
     String ldapSearchBase;
 
-    public LightblueLdapRoleProvider(String server, String searchBase, String bindDn, String bindDNPwd) throws NamingException {
+    public LightblueLdapRoleProvider(String server, String searchBase, String bindDn, String bindDNPwd, Boolean useSSL, String trustStore, String trustStorePassword) throws NamingException {
```

Comment:
Would there be value in keeping both constructors? You wouldn't need to touch LightblueLdapRoleProviderTest then.